src/en/space-station-14/player-interaction/proposals/powered-hardsuits.md

```diff
@@ -0,0 +1,97 @@
+- have little to no downsides to wear
+- can be easily crafted
+- can be loaded up with all sorts of meta equipment
+- require power
+- are basically all the same (due to "modularity" allowing you to put the meta modules in every single suit)
+
+Hardsuits in Space Station 14 are effectively the polar opposite of this, except for the last point. They are all mostly the same, with the only difference between suits being movement speed and armor. What if that wasn't the case though?
+
+## Hardsuits, EVA, and Void suits
+
+Before I get into the cool ideas of what different spacesuits would do, there should be a few specific classifications for spacesuits, since not everything needs abilities.
+
+### EVA suits
+EVA suits would be an abundant and renewable (through cargo) form of space protection. The helmet comes separated and needs to be attached and carried on its own. They provide absolutely zero combat damage resistance. Movement speed and cold protection vary.
+This basically only includes EVA suits, emergency EVA suits, and prisoner EVA suits.
+EVA suits with resistances would be counted as voidsuits.
+
+### Voidsuits
+Voidsuits would basically function as hardsuits without the fancy tech. They would have no abilities, decent movement speed, and not require power. People such as the captain, paramedic, and CMO would get these. The atmos firesuit would sorta fall under this category despite not being called a voidsuit.
+Voidsuits can still have headlamps in the helmet, but these would either use power cells or just have infinite power.
+
+### Hardsuits
+Hardsuits are the real techy stuff, the things cool people get. Everything from the engineer hardsuit to the deathsquad hardsuit would be in this category.
+Hardsuits would be ginormous items, unable to fit in any bag (except for the RD hardsuit), and would give you a movement speed penalty when either carried or worn, with some possibly even taking up two hands. Hardsuits can be turned on when the helmet is deployed, decreasing the movement speed penalty and draining the battery. They should all should do at least SOMETHING different to justify existing, having specific unique abilities that eat up charge faster than only having the suit turned on.
+
+## Suit charge
+
+Currently, the only real downside for wearing a hardsuit 24/7 is a movement speed penalty that needs to stay around 10-30%, or else you'll end up being too slow to do anything and the game will be unfun. This has led to people wearing hardsuits constantly in anticipation of something happening, which isn't interesting at all. If everyone has space protection, what's the point in space?
+
+A powered hardsuit would still slow you down somewhat, you are wearing a bulky metal suit after all. But movement speed is no longer the only downside to wearing hardsuits. As a result, contributors will have more freedom to make certain suits mobile while still having power charge as a balancing factor to make up for it.
```

Comment:
Very good, I like the flexibility we will get when it comes to balance changes. Right now, all balance changes revolving around hardsuits are "if it's fast, make it unarmored, if it's slow, make it armored".

For example, the Warden's suit could get some juicy stuff related to preventing Brig rascals from getting away, or increasing movement speed at the cost of suit charge to chase people (at the cost of obliterating their battery, so they have to put it back in their suit storage locker they get)